content/documentation/pages/1-installation/3-kubernetes/1-helm.md

```diff
@@ -160,6 +146,18 @@ helm install --name my-release \
     --set mysql.mysqlRootPassword=mysqlpwd incubator/spring-cloud-data-flow
 ```
 
+#### Version Compatibility
+
+The following listing shows Spring Cloud Data Flow’s Kubernetes version compatibility with the respective Helm Chart releases:
```

Comment:
> The following listing shows Spring Cloud Data Flow’s version compatibility with the respective Helm Chart releases:

Having Kubernetes in the sentence makes it sound like K8s version, but it is the SCDF version we are referring to here, so let's remove the word.